vyxal/main.py

```diff
@@ -141,7 +141,14 @@ def execute_vyxal(file_name, flags, inputs, output_var=None, online_mode=False):
             print(code + "\n")
 
     ctx.stacks.append(stack)
-    exec(code)
+    try:
+        exec(code)
+    except Exception as e:
+        if ctx.online:
+            ctx.online_output[2] += "\n" + traceback.format_exc()
+            exit(1)
+        else:
+            raise e
```

Comment:
This should just be `raise`, otherwise you get excessive tracebacks (Python thinks `e` was the direct cause of itself; but just `raise` reraises properly).

Reply:
Fixed. Thanks for pointing this out, as I wouldn't have known that.